Simplenote/src/main/res/values/strings.xml

```diff
@@ -43,6 +43,7 @@
     <string name="search">Search notes</string>
     <string name="notes">All Notes</string>
     <string name="trash">Trash</string>
+    <string name="untaggedNotes">Untagged Notes</string>
```

Comment:
Can you please make this `untagged_notes` to match the style of the other string names?